app/views/question_flow/domestic_abuse_applicant.html.slim

```diff
@@ -0,0 +1,24 @@
+- content_for :page_title
+  = t(".question")
+- content_for :back do
+  = back_link(:domestic_abuse_applicant, @check, @back_buttons_invoke_browser_back_behaviour)
+
+.govuk-grid-column-two-thirds
+  = render "shared/heading"
+  = form_for(@form, url: request.path, method: :put) do |form|
+    = form.govuk_error_summary t("generic.error_summary_title")
+
+    h1.govuk-heading-xl = t(".question")
```

Comment:
To be consistent with other screens, it would be better to set this as the `legend` attribute in the govuk_collection_radio_buttons rather than have it as its own element. I believe this affects exactly how classes are applied when there's a validation error.

Reply:
Hey @patrick-laa , I just remembered why I did this. I separated out the heading from radio buttons, in order to follow the design on the ticket. On both of the new designs, there is a gap between heading and the paragraph text.

I did do something like this initially, but it doesn't look like the screen: 

```
- content_for :hint
  p.govuk-boy = t(".paragraph_1")
  p.govuk-boy = t(".paragraph_2")

.govuk-grid-column-two-thirds
  = render "shared/heading"
  = form_for(@form, url: request.path, method: :put) do |form|
    = form.govuk_error_summary t("generic.error_summary_title")

    = form.govuk_collection_radio_buttons :domestic_abuse_applicant,
            yes_no_options, :first, :last,
            legend: { text: t(".question"), size: "xl", tag: "h1" },
            hint: -> { content_for(:hint) }
```

Would I add some stylings to create a gap??

Reply:
Ah ok. So that gap is present on the level of help screen, so you could copy the approach there (i.e. keep legend as `nil`, but instead of having a manual `h1`, pass in the heading as a `header_text` variable to the `shared/heading` partial). But also, I think I'm going to ask about this in the UCD channel, because looking at it we're already wildly inconsistent with this stuff and I think the new designs add yet more inconsistencies. It might be that if I point that out, the designs will change to something more standard.

Reply:
I've added in some spacing. 👍🏽